### DISCUSSION TITLE
How to skip a rule in Terraform?

### DISCUSSION THREAD

**Discussion by gtirloni** [Checkmarx/kics#5671] [Q&A] [answered]
If I have a Terraform resource that triggers a Kics rule and I'm sure it can be skipped. Is it possible to specify it somehow in the .tf file?

For example, if I have a rule that says GCP storage buckets must have versioning enabled, but I don't want to enable it for a specific bucket. How can I instruct Kics to ignore that rule for that specific bucket definition?

**Comment by cx-nuno-oliveira** [comment, marked as answer]
Hi @gtirloni 
I believe this is exactly what you're looking for: 
https://docs.kics.io/latest/running-kics/#using_commands_on_scanned_files_as_comments

Let us know if this helps :)
